PlaceRouteHierFlow/router/PowerRouter.cpp

```diff
@@ -23,10 +23,10 @@ PowerRouter::PowerRouter(PnRDB::hierNode& node, PnRDB::Drc_info& drc_info, int L
      CreatePowerGrid(node, drc_info, Lmetal, Hmetal, rate);
      std::cout<<"CheckPoint 2"<<std::endl;
      Physical_metal_via_power_grid(Vdd_grid);
-     Vdd_grid.name = "vss";
+     Vdd_grid.name = "vdd";
```

Comment:
@Lastdayends Changed back to names that make sense. What if the user changed the names? Where can we find that mapping? (We can fix this in a new PR.)